upgrade.sh

```diff
@@ -12,7 +12,7 @@ ImageResourceGroup=images ImageResourceName=centos7-3.10-201806231427 \
 
 # poor man's helm (without tiller running)
 helm template pkg/helm/chart -f _data/_out/values.yaml --output-dir _data/_out
-oc apply -Rf _data/_out/osa/templates
+KUBECONFIG=aks/admin.kubeconfig kubectl apply -n $RESOURCEGROUP -Rf _data/_out/osa/templates
```

Comment:
Not sure if `kubectl apply` works correctly when you have created the objects with `kubectl create`. I think `apply` needs to own creation too.

Reply:
+1 will fix